storage/ndb/include/kernel/Interpreter.hpp

```diff
@@ -378,10 +812,11 @@ Interpreter::ExitLastOK(){
   return EXIT_OK_LAST;
 }
 
+/* Opcode is bit 0-5 and bit 15, giving a range of 0-127 */
 inline
 Uint32
 Interpreter::getOpCode(Uint32 op){
-  return op & 0x3f;
+  return (op & 0x3f) + (((op >> 15) & 1) << 6);
```

Comment:
Why not `return (op & 0x3f) + ((op >> 9) & 0x40);`?

Reply:
Or actually, perhaps even faster, `return op & 0x803f;` and then use constant values `0x8000` through `0x803f` for the opcodes